src/citrine/__init__.py

```diff
@@ -1,2 +1,5 @@
 # TODO: Add a docstring here
 from citrine.citrine import Citrine  # noqa: F401
+import logging
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
This establishes `INFO` as the default log level. A somewhat perplexing side-effect is the inclusion of the module name in the output, which is actually very nice for locating the logger. We want want to add more deliberate configuration here down the road.